client/src/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Title from './components/title.jsx';
+import Price from './components/price.jsx';
+import ReviewScore from './components/reviewScore.jsx';
+import Questions from './components/questions.jsx';
+import QuantitySelector from './components/quantitySelector.jsx';
+import ColorSelector from './components/colorSelector.jsx';
+import PictureCarousel from './components/pictureCarousel.jsx';
+import PictureCarouselEntry from './components/pictureCarouselEntry.jsx';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  // async componentDidMount() {
+  //   await $.ajax({
+
+  //   })
+  // }
+
+  render() {
+    return (
+      <div>
+        <Title />
+        <Price />
+        <div>
+          <ReviewScore />
+          <span>
+            <Questions />
+            <QuantitySelector />
+          </span>
+          <div>
+            <ColorSelector />
+            <div>
+              <PictureCarousel />
```

Comment:
You already have PictureCarousel rendered in the ColorSelector. Cut it either here or there (I'd recommend cutting it here) so that you don't have it twice.

Reply:
Check the other components to be sure you're not putting them twice!